misc/sample_application/image_builder/create_image.sh

```diff
@@ -152,6 +152,25 @@ EOF
         sudo chown root:root /etc/schroot/chroot.d/acrn-guest
 }
 
+function create_uio_config() {
+    local mount_point=$1
+    local temp_file=$(mktemp /tmp/rc.local.XXXX)
+
+    cat << EOF > ${temp_file}
+modprobe uio
+modprobe uio_pci_generic
+
+for i in {0..3}
+do
+bash -c 'echo "1af4 1110" > /sys/bus/pci/drivers/uio_pci_generic/new_id'
+sleep 1
+done
+EOF
+
+    sudo mv ${temp_file} /etc/rc.local && \
```

Comment:
I'm assuming we are not overwriting any existing `/etc/rc.local` here, or are we?